tools/logos_check.py

```diff
@@ -0,0 +1,75 @@
+#!/usr/bin/env python3
+
+import sys
+import argparse
+from pathlib import Path
+
+from PIL import Image
+
+
+class ImageCheck():
+    def __init__(self, imgpath: Path) -> None:
+        self.imgpath = imgpath
+        self.image = Image.open(imgpath)
+
+        self.fails: list[str] = []
+
+    def check(self) -> bool:
+        self.fails.clear()
+        for fn_name in dir(self):
+            if fn_name.startswith("check_"):
+                getattr(self, fn_name)()
+
+        if not self.fails:
+            return True
+
+        print(f"Image '{self.imgpath}' failed tests:")
+        for fail in self.fails:
+            print(f"  - {fail}")
+        print()
+        return False
+
+    def check_type(self) -> None:
+        format = self.image.format.lower()
+        accepted_formats = ["png", "jpeg", "webp"]
```

Comment:
Should we accept all those formats? Many software don't support web yet. And JPG doesn't allow background transparency which is nice for dark/light background. I think only PNG should be supported, but we could discuss it on the issue.

Reply:
(currently we have `.png` exclusively in `logos/` indeed)